micromanager_gui/prop_browser.py

```diff
@@ -226,9 +226,9 @@ def _on_le_change(self, e):
 
 
 if __name__ == "__main__":
-    from micromanager_gui._core._mmcore_plus import MMCorePlus
+    from pymmcore_plus import CMMCorePlus
 
```

Comment:
Maybe should be `RemoteMMCore` ?

Reply:
perhaps, but lets actually leave this out of this PR altogether.  @fdrgsp  can you revert any changes made to the prop_browser?  let's deal with those in a follow up PR